tools/RunSigningJob.ps1

```diff
@@ -0,0 +1,22 @@
+$bypassPackaging = $env:APPVEYOR_PULL_REQUEST_NUMBER -and -not $env:APPVEYOR_PULL_REQUEST_TITLE.Contains("[pack]")
+$directoryPath = Split-Path $MyInvocation.MyCommand.Path -Parent
+
+if (-not $bypassPackaging) {
+  # Only sign the ExtensionsMetadataGenerator
+  New-Item -ItemType Directory -Force -Path "$directoryPath\..\buildoutput\signing"
+  Compress-Archive $directoryPath\..\buildoutput\Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator*.nupkg $directoryPath\..\buildoutput\signing\tosign.zip  
+  Remove-Item $directoryPath\..\buildoutput\Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator*.nupkg
+
+  if ($env:SkipAssemblySigning -eq "true") {
+    "Signing disabled. Skipping signing process."
+    exit 0;
+  }
+
+  $ctx = New-AzureStorageContext $env:FILES_ACCOUNT_NAME $env:FILES_ACCOUNT_KEY
+  Set-AzureStorageBlobContent "$directoryPath/../buildoutput/signing/tosign.zip" "azure-functions-host" -Blob "$env:APPVEYOR_BUILD_VERSION.zip" -Context $ctx
+
+  $queue = Get-AzureStorageQueue "signing-jobs" -Context $ctx
+
+  $messageBody = "SignNuget;azure-functions-host;$env:APPVEYOR_BUILD_VERSION.zip"
```

Comment:
This is for package signing only. These assemblies should probably be strong name signed as well (that will cause an identity change, but it should be OK for these  files)

Reply:
See my comment here: https://github.com/Azure/azure-functions-host/pull/4071#issuecomment-462818459. 

We'll need to change the signing process to work here since the package has a bunch more assemblies than just ours. I'll talk with @mhoeger and take a look at that script right now.